api/clients/disperser_client_v2.go

```diff
@@ -60,7 +60,7 @@ var _ DisperserClientV2 = &disperserClientV2{}
 //
 //	// Subsequent calls will use the existing connection
 //	status2, blobKey2, err := client.DisperseBlob(ctx, data, blobHeader)
-func NewDisperserClientV2(config *DisperserClientV2Config, signer corev2.BlobRequestSigner, prover encoding.Prover) (*disperserClientV2, error) {
+func NewDisperserClientV2(config *DisperserClientV2Config, signer corev2.BlobRequestSigner, prover encoding.Prover, accountant Accountant) (*disperserClientV2, error) {
```

Comment:
It's quite difficult to construct a new Accountant. Does it make sense to provide a helper method that creates an accountant from the `GetPaymentState` result? 

Let's also add a way to call `GetPaymentState` in this client

Reply:
added a helper function `PopulateAccountant` that should run after creating `NewDisperserClientV2` by constructing accountant from `GetPaymentState`

What do you think about allowing the user of DisperserClientV2 to provide their own accountant, or simply passing in `&accountant{}` to `NewDisperserClientV2` and then explicitly calling `PopulateAccountant` to fill in the accountant state? ([5004cf9](https://github.com/Layr-Labs/eigenda/pull/928/commits/5004cf9ad401f641afce86088fd2633d1b947f87))

Reply:
The reason is `GetPaymentState` is a function of `disperserClientV2`. If we write a standalone `PopulateAccountant`, then we would need it to be able to `initOnceGrpcConnection`, pass in a signer for creating the request, and a `disperser_rpc.DisperserClient` that makes the actual request. I figured it is cleaner to just use disperserClientV2